services/gcp/database/redis.yml

```diff
@@ -0,0 +1,29 @@
+name: Redis Instances
+description: >-
+  Redis instances in Google Cloud Platform (GCP) provide an in-memory data store that can be used for caching and high-performance data processing. Redis instances store data in key-value pairs and support various data types, including strings, lists, sets, and hashes. Redis instances are used to store and retrieve data rapidly and can be accessed by applications running on GCP or external systems.
+scope: CRITICAL
```

Comment:
IIUC, the typical use case of Redis is as a cache or buffer. In this case the scope would be HIGH rather than CRITICAL, as durable data can not be deleted. (And, I believe, data can not be read or injected using any of the APIs here).